content/docs/command-reference/exp/gc.md

```diff
@@ -40,9 +40,10 @@ separately to delete it.
   the workspace (implies `-w`). Note that both options can be combined, for
   example using the `-aT` flag.
 
-- `--all-commits` - same as `-a` or `-T` above, but applies to _all_ Git commits
-  as well as the workspace (implies `-w`). This is mainly only needed when
-  clearing the experiments run queue.
+- `--all-commits` - same as `-a` or `-T` above, but applies to _all_
+  [reachable](https://git-scm.com/docs/gitglossary#def_reachable) Git commits as
+  well as the workspace (implies `-w`). This is mainly needed when clearing the
+  experiments run queue.
```

Comment:
> Continuation of https://github.com/iterative/dvc.org/pull/2300#pullrequestreview-617325868

Linking to "reachable" to match the details added to `exp remove`. Notice it's vague enough that it can mean what we want it to mean, I think (since we don't specify reachable from where):

![image](https://user-images.githubusercontent.com/1477535/112094391-c8fbd280-8b60-11eb-8417-5bab4dbc384a.png)

Also, keeping just "all commit" seems incorrect, esp. in the context of experiments which are commits (fact which we explain in docs) but aren't affected by this option.

Reply:
p.s. on "reachable commits": would "regular commits" work better? Or "your commits"?

Note that this change has already been applied to `exp show -A` (accidentally https://github.com/iterative/dvc.org/pull/2300#pullrequestreview-619179757).

Reply:
p.s. And if we keep "regular" or other term, should it be applied to all DVC commands with `--all-commits`? There's another 5-6 I think.